packages/notifi-wallet-provider/lib/types.ts

```diff
@@ -20,17 +20,25 @@ export type WalletKeysBase = {
   bech32: string;
   base64: string;
   hex: string;
+  grantee: string;
```

Comment:
I think we still need to know about the format about the grantee so that we can reuse this field for those upcoming chains potentially.
So, If we can not get to know it now, we might want to put a todo item, something like below.
`//TODO: specifically for Xion now, make it generic pattern, checking the format`